src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderAssessmentRepository.java

```diff
@@ -10,21 +10,12 @@
 public interface OffenderAssessmentRepository extends CrudRepository<OffenderAssessment, OffenderAssessment.Pk> {
     Optional<OffenderAssessment> findByBookingIdAndAssessmentSeq(Long bookingId, Integer assessmentSeq);
 
-    @Query("""
-        SELECT oa FROM OffenderAssessment oa 
-        INNER JOIN oa.offenderBooking booking INNER JOIN booking.offender offender 
-        INNER JOIN oa.assessmentType assessment
-        WHERE offender.nomsId = :offenderNo AND assessment.cellSharingAlertFlag = 'Y'
-        ORDER BY oa.assessmentDate DESC
-        """)
-    List<OffenderAssessment> findByCsraAssessmentAndByOffenderNo_OrderByLatestFirst(String offenderNo);
-
     @Query("""
         SELECT oa FROM OffenderAssessment oa 
         INNER JOIN oa.offenderBooking booking INNER JOIN booking.offender offender 
         INNER JOIN oa.assessmentType assessment
```

Comment:
minor point but don't think you need the word INNER

Reply:
True - I'll sort it later as i need to get this fix out, and with CircleCI being slow it will take a while to push the fix